src/tree_cmd.cpp

```diff
@@ -371,27 +371,27 @@ void GenerateTrees()
 
 /**
  * Plant a tree.
- * @param tile end tile of area-drag
+ * @param end_tile end tile of area-drag
  * @param flags type of operation
  * @param p1 tree type, TREE_INVALID means random.
  * @param p2 start tile of area-drag of tree plantation
  * @param text unused
  * @return the cost of this operation or an error
  */
-CommandCost CmdPlantTree(TileIndex tile, DoCommandFlag flags, uint32 p1, uint32 p2, const char *text)
+CommandCost CmdPlantTree(TileIndex end_tile, DoCommandFlag flags, uint32 p1, uint32 start_tile, const char *text)
```

Comment:
See earlier comment about renaming `p2`. Either way, you forgot doxygen in this case :)